internal/background/initialize.go

```diff
@@ -13,11 +13,22 @@ import (
 // incoming requests to overload the sender.
 const availabilityStatusBatchSize = 1024
 
-// Initialize starts background goroutines. Use context cancellation to stop them.
-func Initialize(ctx context.Context) {
+// InitializeApi starts background goroutines for REST API processes.
+// Use context cancellation to stop them.
+func InitializeApi(ctx context.Context) {
 	logger := ctxval.Logger(ctx).With().Bool("background", true).Logger()
 	ctx = ctxval.WithLogger(ctx, &logger)
 
 	// start availability request batch sender
 	go sendAvailabilityRequestMessages(ctx, availabilityStatusBatchSize, 5*time.Second)
 }
+
+// InitializeWorker starts background goroutines for worker processes.
+// Use context cancellation to stop them.
+func InitializeWorker(ctx context.Context, workerName string) {
+	logger := ctxval.Logger(ctx).With().Bool("background", true).Logger()
+	ctx = ctxval.WithLogger(ctx, &logger)
+
+	// start job queue telemetry
+	go jobQueueMetricLoop(ctx, 30*time.Second, workerName)
+}
```

Comment:
Background package now have two functions:

* InitializeApi - for REST API processes
* InitializeWorkers - for worker processes

If you need any background goroutine, add it here.